src/main/java/com/gymrats/service/CalculatorService.java

```diff
@@ -1,16 +1,35 @@
 package com.gymrats.service;
 
-import com.gymrats.model.BmiNotLoggedRequest;
-import com.gymrats.model.BmiNotLoggedResponse;
+import com.gymrats.model.*;
 import org.springframework.stereotype.Service;
 
+import java.time.LocalDate;
+
 @Service
 public class CalculatorService {
 
     public BmiNotLoggedResponse calculateBmi(BmiNotLoggedRequest requestBmi) {
         float weight = requestBmi.weight();
-        float height = requestBmi.height();
+        float height = requestBmi.height()/100;
         float bmi = weight / (height * height);
         return new BmiNotLoggedResponse(bmi);
     }
+
+    public BmrNotLoggedResponse calculateBmr(BmrNotLoggedRequest requestDataBmr){
+        int yearOfBirth = requestDataBmr.dateOfBirth().getYear();
+        LocalDate currentDate = LocalDate.now();
+        int currentYear = currentDate.getYear();
+        int userAge = currentYear - yearOfBirth;
+        GenderEnum sex = requestDataBmr.sex();
+        float weight = requestDataBmr.weight();
+        float height = requestDataBmr.height();
+        float bmr;
+
+        if(sex.equals(GenderEnum.WOMAN)){
+            bmr = (float)(447.593 + (9.247 * weight) + (3.098 * height) - (4.330 * userAge));
+        } else {
```

Comment:
I don't like that in the first if you check GenderEnum.Woman and then you put the man in else, it doesn't look legible. I prefer else if (sex.equals(GenderEnum.Man) {}